.github/workflows/build-extensions.yml

```diff
@@ -223,9 +223,12 @@ jobs:
             system: manylinux2014 x86_64
             host_runner: ubuntu-latest
             docker_tag: manylinux2014_x86_64
+            test_path: test/plugins/wasi_nn
+            test_bin: wasiNNTests
             output_path: plugins/wasi_nn
             output_bin: libwasmedgePluginWasiNN.so
             build_type: Release
+            test: false
```

Comment:
I think it's not necessary to test on the manylinux now.
We can refactor the CI in the future and add the test.

Reply:
The test of PyTorch is only on `ubuntu`, only tflite tests on the `manylinux`